.circleci/config.yml

```diff
@@ -6,7 +6,6 @@ aliases:
     - kube_1_13: &kube_1_13 "1.13.7"
     - kube_1_14: &kube_1_14 "1.14.6"
     - kube_1_15: &kube_1_15 "1.15.0"
-  kind_version: &kind_version "0.5.1"
```

Comment:
Do we not want to keep this as a variable at the top of the file?

Reply:
right, need to investigate passing variable in circle-ci. First time I tried it resulted in some unset value

Reply:
I moved kind installation to Makefile as this can be useful as well for contributors